base/loading.jl

```diff
@@ -3053,24 +3053,26 @@ global parse_pidfile_hook
 # the same package cannot be precompiled from different projects and/or different preferences at the same time.
 compilecache_pidfile_path(pkg::PkgId) = compilecache_path(pkg, UInt64(0); project="") * ".pidfile"
 
+const compilecache_pidlock_stale_age = 10
+
 # Allows processes to wait if another process is precompiling a given source already.
 # The lock file mtime will be updated when held every `stale_age/2` seconds.
```

Comment:
This isn't a very accurate claim since most systems are not a realtime kernel and Julia is not preemptive. A more accurate claim, to reflect observed reality and possible execution times, would be to say it updates at most every stale_age/2, with expected variance of 10 seconds or more being infrequent but not unusual.